docs/GuideTVOS.md

```diff
@@ -0,0 +1,89 @@
+---
+id: guide-tvos
+title: Apple TV changes
+layout: docs
+category: Guides (Apple TV)
+permalink: docs/guide-tvos.html
+banner: ejected
+next: native-modules-android
+previous: communication-ios
+---
+
+Apple TV support has been implemented with the intention of making existing React Native iOS applications "just work" on tvOS, with few or no changes needed in the Javascript code for the applications.
+
+The UIExplorer example project supports Apple TV; use the `UIExplorer-tvOS` build target to build for tvOS.
+
+## Build changes
+
+- *Native layer*: React Native Xcode projects all now have Apple TV build targets, with names ending in the string '-tvOS'.
+
+- *react-native init*: New React Native projects created with `react-native init` will have Apple TV target automatically created in their XCode projects.
+
+- *Javascript layer*: Support for Apple TV has been added to `Platform.ios.js`.  You can check whether code is running on AppleTV by doing
+
+```js
+var Platform = require('Platform');
+var running_on_apple_tv = Platform.isTVOS;
+```
+
+## Code changes
+
+- *General support for tvOS*: Apple TV specific changes in native code are all wrapped by the TARGET_OS_TV define.  These include changes to suppress APIs that are not supported on tvOS (e.g. web views, sliders, switches, status bar, etc.), and changes to support user input from the TV remote or keyboard.
+
+- *Common codebase*:  Since tvOS and iOS share most Objective-C and Javascript code in common, most documentation for iOS applies equally to tvOS.
+
+- *Access to touchable controls*: When running on Apple TV, the native view class is `RCTTVView`, which has additional methods to make use of the tvOS focus engine.  The `Touchable` mixin has code added to detect focus changes and use existing methods to style the components properly and initiate the proper actions when the view is selected using the TV remote, so `TouchableHighlight` and `TouchableOpacity` will "just work".  In particular:
+  - `touchableHandleActivePressIn` will be executed when the touchable view goes into focus
```

Comment:
This may not render correctly as a list without a blank space between the list and the preceding paragraph.